src/components/Navigator/Navigator.tsx

```diff
@@ -39,9 +39,9 @@ export const Navigator: React.FC<NavigatorProps> = ({
       value={activeResource}
     >
       <List aria-label="select chapter" data-testid="navigator-list">
-        {resources.map((resource) => (
-          <Trigger value={resource.id as string}>
-            {getLabel(resource.label as InternationalString, "en")}
+        {resources.map(({ id, label }) => (
+          <Trigger key={id} value={id as string}>
```

Comment:
Looks like we both put in a fix here. I'll let you decide how to handle. Yours is de-structured so I lean that way.

Reply:
Cool.  Let's keep the destructured approach.  Easier to read maybe?